### PR TITLE
Add support for Symfony 5 and 6

### LINKED ISSUE #1453: Roadmap SonataAdmin 4 / Symfony 5
## Feature Request

Hey,

is there a roadmap or a plan, when its possible to use the SonataUserBundle with the actual SonataAdmin v4 and Symfony v5.3? 

Thanks!

### PR DESCRIPTION
**Pull request by jordisala1991**
## Subject

I am targeting this branch, because this can only happen after drop of FOSUserBundle.

Closes #1453 .

## Changelog

```markdown
### Added
- Added support for Symfony 5.
- Added support for Symfony 6.
```

### CODE REVIEW

**Comment by jordisala1991** [comment]
This is blocked until #1449 is merged, but you can start testing it with Symfony 5 (Symfony 6 misses a release for `symfony/acl`) using the branch of my fork @Hanmac

File: src/Action/LoginAction.php
Comment (Hanmac) [inline src/Action/LoginAction.php:104]: can AuthenticationUtils be used there? the other bundle, NucleosUserAdminBundle has a fallback in case AuthenticationUtils doesn't exist, but it exist since  2.6 ?

https://github.com/nucleos/NucleosUserAdminBundle/blob/2.3.x/src/Action/LoginAction.php#L152-L176

Comment (jordisala1991) [reply src/Action/LoginAction.php:104]: Yes, it can be used, but since it does not seem needed to make all builds pass I prefer to do it on another Pr, can you raise an issue about it so we dont forget?

Comment (jordisala1991) [reply src/Action/LoginAction.php:104]: It will be done on: https://github.com/sonata-project/SonataUserBundle/pull/1457

**Review by VincentLanglet — APPROVED** [review]

**Pull request merged**
